.header.md

```diff
@@ -11,8 +9,9 @@ This repo template is used to seed Terraform Module templates for the [AWS I&A G
     - [tfsec](https://aquasecurity.github.io/tfsec/v1.0.11/)
     - [terraform-docs](https://github.com/terraform-docs/terraform-docs)
     - [golang](https://go.dev/doc/install) (for macos you can use `brew`)
+    - [coreutils](https://www.gnu.org/software/coreutils/)
 
-Write code according to [I&A module standards](https://github.com/aws-ia/standards-terraform/content/standards/index.en.md) (need to update link once it goes live)
+Write code according to [I&A module standards](https://github.com/aws-ia/standards-terraform/blob/main/content/_index.en.md)
```

Comment:
Please make this link: https://aws-ia.github.io/standards-terraform/

Reply:
Added the updated url